cwls/bam2bedgraph.cwl

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env cwl-runner
+
+doc: |
+  ![build_status](https://quay.io/repository/wtsicgp/dockstore-cgpbigwig/status)
+
+  A wrapper for the cgpBigWig bam2bedgraph tool.
+
+  bam2bedgraph documentation can be found [here](https://github.com/cancerit/cgpBigWig#bam2bedgraph)
+
+  In order to run the example found in `example/bam2bedgraph.json` please download the relevant reference files
+  listed in the [`README`](https://github.com/cancerit/cgpdockstore-cgpbigwig/README.md#Example_reference_files)
+
+  See the [dockstore-cgpbigwig](https://github.com/cancerit/dockstore-cgpbigwig)
+  website for more information about this wrapper.
+
+  For queries relating to the underlying software see [cgpBigWig](https://github.com/cancerit/cgpBigWig).
+
+cwlVersion: v1.0
+
+class: CommandLineTool
+
+id: "cgpbigwig-bam2bedgraph"
+label: "cgpbigwig-bam2bedgraph"
+
+baseCommand: bam2bedgraph
+
+requirements:
+  - class: DockerRequirement
+    dockerPull: "drjsanger/randomtesting:cgpbigwig01"
```

Comment:
Don't forget to update in release branch.  FYI the dockstore prerelease version I'm testing (1.3.4) is working with `$mixin` syntax so you'll soon be able to move this into a `mixins/requirements.yml` file so that all descriptors are updated by changing a single file.  I'm going to recommend we hold off release until this is available on the main site (currently only supported on staging).